recipe/migrations/r-base43.yaml

```diff
@@ -0,0 +1,15 @@
+migrator_ts: 1682187595
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+  operation: key_add
+  commit_message: "Rebuild for r-base 4.3"
+  primary_key: r_base
+  automerge: True
+  longterm: True
+  include_noarch: True
+  pr_limit: 4
```

Comment:
```suggestion
  pr_limit: 4
  conda_forge_yml_patches:
    build_platform.win_64: disabled
```

Reply:
Yeah, I don't know much about migrators... Didn't even know https://github.com/regro/cf-scripts/pull/992 existed :).

I'd assume this works because `win_disabled` is no known CI provider and as such nothing gets rendered for that platform?

Also, does this mean the answer to https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4363#discussion_r1181866579 :
> When dropping 4.1, we need to skip windows entirely. Not sure how to do that.

is that `conda_forge_yml_patches` works fine in combination with the rest of the migration (`operation: key_add`)?

Reply:
Given the discussion from
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/592/files#r418585834
and
https://github.com/regro/cf-scripts/blob/dec7247bdc7b9e2bf85eb38e0870be741edaab2e/conda_forge_tick/migrators/migration_yaml.py#L75
it looks like this (intentionally) would not work for feedstocks where `provider: win` is not set already, correct?

Reply:
> is that conda_forge_yml_patches works fine in combination with the rest of the migration (operation: key_add)?

It should, but not sure if `key_add` is the best choice here. If we do `key_add`, then feedstocks will be in a un-rerenderable state until the migration PR is merged unless `skip: True # [win]` is added.

>  it looks like this (intentionally) would not work for feedstocks where provider: win is not set already, correct?

`provider: win` is available by default.

Reply:
> I'd assume this works because win_disabled is no known CI provider and as such nothing gets rendered for that platform?

yes